FocaAI/FocaAI-chrome-extension
# Chore/shadcnui

## Description


This PR configures Shadcnui with a theme provider, which allows us to configure the theme via CSS variables and set up dark mode and other themes.

It also adds types to use the API Prompt from Google types. 
## Related Tickets & Documents

Resolve #5 
## Mobile & Desktop Screenshots/Recordings
![image](https://github.com/user-attachments/assets/1a1bba61-34bc-4609-950a-4963811d379f)



## Steps to QA

1. After following the setup and installation from [README.md](https://github.com/FocaAI/FocaAI-chrome-extension/blob/main/README.md)
2. Go to the extensions list then press the right button of the mouse in the extension icon, then choose the open side Panel option
3. see the content 
## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?
![funny-cat-dark-mode](https://github.com/user-attachments/assets/03497c6c-15f5-4839-aa5a-df10da7decbd)
